tests/api-mongo.test.js

```diff
@@ -608,3 +609,36 @@ describe('MongoDB Words', () => {
     });
   });
 });
+
+describe('API Requests For Home Directory "/"', () => {
+  it('should return response status of 404 in /undefinedRoute', (done) => {
+    const route = '/undefinedRoute';
+    getAPIUrlRoute(route)
+      .end((_, res) => {
+        expect(res.status).to.equal(404);
+        done();
+      });
+  });
+
+  it('should contain Igbo API in / route', (done) => {
+    getAPIUrlRoute('/')
+      .end((_, res) => {
+        expect(res.status).to.equal(200);
+        expect(res.body).to.be.an('object');
+        expect(res.text).to.contain('Igbo API');
+        done();
+      });
+  });
+
+  it('/api/v1/words should get words document', (done) => {
```

Comment:
Why did you add these tests? If you look in `api-mongo.test.js` you can see that we are testing the get words route.

Reply:
This one is getting data from the hosted platform